lua/autorun/client/cl_votestuff_init.lua

```diff
@@ -0,0 +1,73 @@
+if SERVER then return end;
+
+local filterPattern = "&#?[%w]+;"
+
+surface.CreateFont( "Question-font", {
+	font = "Century Gothic",
+	size = ScreenScale(7),
+	scanlines = 0,
```

Comment:
generally you shouldn't specify parameters that match the default values, and certainly not for this